etc/kayobe/environments/aufn-ceph/inventory/groups

```diff
@@ -0,0 +1,29 @@
+# Kayobe groups inventory file. This file should generally not be modified.
+# It declares the top-level groups and sub-groups.
+
+###############################################################################
+# Ceph groups
+
+[storage-ceph]
+storage0
+storage1
+storage2
```

Comment:
We shouldn't really have hosts in the groups file. This should get generated via the group map in overcloud.yml